### DISCUSSION TITLE
Daily 5/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#95] [General]
Armé un diseño simple para el listado de compras, y lo implementé también muy simplemente en HTML. Creé el directorio images para que puedan poner las imágenes dentro, además agregué una imagen de Wikipedia que se puede usar como placeholder cuando necesiten una imagen y no tengan para no bajar imágenes con copyright (solo hay que colocar un disclaimer porque es una imagen CC-SA pero si solo es temporal se la puede usar así y después cambiar por la imagen final). Hoy pienso tomar alguna otra página para hacer, y tal vez empezar con el CSS (aunque tal vez lo haga directamente en Bootstrap). También quiero armar un header o los esqueletos de todos los archivos vacíos para que todos tengan el mismo header y footer al menos, aunque no estoy seguro si voy a tener tiempo hoy para hacerlo.

**Comment by NataliaAlvarezIspc** [comment]
Ayer solo pude hacer un pequeño repaso de SGC. 
Hoy estoy dandole forma en HTML al formulario de registración y tratare de hacer tambien el de login.

**Comment by AhumadaCeleste** [comment]
Ayer viaje por trabajo y llegue muy tarde. El lunes retome Cisco. También comencé a armar de a poco las tareas asignadas del proyecto y repase la primera parte de SGC. Hoy presenciaré la clase de Ingles a las 18. También continuo construyendo lo del proyecto para terminarlo lo antes posible y continuar con  de SGC y Cisco el fin de semana. Mañana espero poder concluir lo que inicie hoy con  respecto al proyecto.

**Comment by Brian-Almada** [comment]
Ayer escuché la última clase de SGC en donde se enseñó sobre virtualización. Luego me puse a ver lo del html de la issue en la que trabajo. Hoy voy a terminar con eso y con lo que me quede de tiempo voy a dar una sobre leída a SGC para recordar conceptos y a hacer algunos ejercicios de Python para pulir un poco ese lenguaje.
